libpretixsync/src/main/java/eu/pretix/libpretixsync/sync/BaseDownloadSyncAdapter.java

```diff
@@ -12,6 +12,11 @@
 import java.util.Map;
 import java.util.Set;
 import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import java8.util.concurrent.CompletableFuture;
```

Comment:
Does this absolutely have to be `java8...`? `java...` seems to working just fine - actually it seems to be only working for me when not using `java8`...